src/widgets/Tabs/PatientSelect.js

```diff
@@ -49,89 +55,50 @@ import { generateUUID } from '../../database/index';
-        />
-      );
-    },
-    [patients]
-  );
-
-  const renderHeader = React.useCallback(
-    () => (
-      <DataTableHeaderRow
-        columns={columns}
-        isAscending={isAscending}
-        sortKey={sortKey}
-        onPress={onSortData}
-      />
-    ),
-    [columns, sortKey, isAscending]
-  );
+  const keyboardIsOpen = useKeyboardIsOpen();
 
   return (
-    <FlexView style={pageTopViewContainer}>
-      <AfterInteractions placeholder={null}>
-        <FlexRow style={{ marginBottom: 7 }} justifyContent="flex-end">
-          <VaccinePrescriptionInfo />
-        </FlexRow>
-
-        <View style={localStyles.container}>
-          <View style={localStyles.formContainer}>
-            <FormControl
-              inputConfig={formConfig}
-              onSave={onFilterData}
-              showCancelButton={false}
-              saveButtonText={generalStrings.search}
-            />
-          </View>
-          <View style={localStyles.verticalSeparator} />
-          <View style={localStyles.listContainer}>
-            <DataTable
-              data={patients}
-              columns={columns}
-              renderHeader={renderHeader}
-              renderRow={renderRow}
-              keyExtractor={item => item.id}
-              getItemLayout={getItemLayout}
-            />
+    <FlexView style={{ ...pageTopViewContainer, paddingVertical: 0 }}>
+      <Paper
+        style={{ flex: 6 }}
+        contentContainerStyle={{ flex: 1 }}
+        headerText={vaccineStrings.vaccine_dispense_step_one_title}
+      >
+        <AfterInteractions placeholder={null}>
+          <View style={localStyles.container}>
+            <View style={localStyles.formContainer}>
+              <FormControl
+                inputConfig={formConfig}
+                onSave={onFilterData}
+                showCancelButton={false}
+                saveButtonText={generalStrings.search}
+                showSaveButton={false}
+              />
+            </View>
+
+            <View style={localStyles.listContainer}>
+              <SimpleTable selectRow={selectPatient} data={patients} columns={columns} />
+            </View>
           </View>
-        </View>
-
-        <FlexRow justifyContent="flex-end" alignItems="flex-end">
-          <PageButtonWithOnePress
-            text={buttonStrings.cancel}
-            onPress={onCancelPrescription}
-            style={{ marginRight: 7 }}
-          />
+        </AfterInteractions>
+      </Paper>
+      {!keyboardIsOpen && (
+        <FlexRow flex={0} style={{ bottom: 0 }} justifyContent="flex-end" alignItems="flex-end">
+          <PageButtonWithOnePress text={buttonStrings.cancel} onPress={onCancelPrescription} />
           <PageButton
             text={`${dispensingStrings.new} ${dispensingStrings.patient}`}
             onPress={createPatient}
-            style={{ marginLeft: 5 }}
+            style={{ marginLeft: 'auto' }}
           />
         </FlexRow>
-      </AfterInteractions>
+      )}
```

Comment:
I *think* the buttons are good like this, but not sure!

Could maybe put the `New Patient`  as a button in the `Paper` header

Reply:
I think the `New patient` should be up in the header - it's just that there was more space at the bottom!
The problem with the bottom is that this is where 'confirm' / 'next' are.. and on this form, selecting a patient is 'next'